### DISCUSSION TITLE
Camera Barcode Scanning on Android

### DISCUSSION THREAD

**Discussion by No1e** [dotnet/maui#15243] [General]
Hi guys,

I wonder which library do you use for camera barcode scanning?

Best regards,
Nole

**Comment by acaliaro** [comment]
@jfversluis do some videos about this

https://www.youtube.com/@jfversluis/videos

**Reply by jfversluis** [reply]
Thanks Alessandro! Here is the videos in no particular order.

* ZXing.Net.Maui: https://youtu.be/ostgj2xB_ok
* Google Vision ML Kit: https://youtu.be/WWP2t-B5ADU
* Camera.MAUI: https://youtu.be/FuvFrIS9wm0
